doc/classes/CollisionPolygon2D.xml

```diff
@@ -17,9 +17,12 @@
 			If [code]true[/code], no collisions will be detected.
 		</member>
 		<member name="one_way_collision" type="bool" setter="set_one_way_collision" getter="is_one_way_collision_enabled" default="false">
-			If [code]true[/code], only edges that face up, relative to [CollisionPolygon2D]'s rotation, will collide with other objects.
+			If [code]true[/code], only edges that face [member one_way_collision_direction], relative to [CollisionPolygon2D]'s rotation, will collide with other objects.
```

Comment:
This is a bit misleading. The default collision direction is down.

Reply:
I've opted to just mention this member in a note.